config/crds/patches/kustomization.yaml

```diff
@@ -2,21 +2,43 @@ bases:
   - ../bases
 
 patchesJson6902:
+  # patches that apply to all CRDS
   - target:
       group: apiextensions.k8s.io
       version: v1beta1
       kind: CustomResourceDefinition
       name: apmservers.apm.k8s.elastic.co
-    path: apm-and-kibana-podtemplate-patch.yaml
+    path: all-patches.yaml
```

Comment:
sound like it contains all the patches :-)

Reply:
I agree. Would `base-patches` or `generic-patches` be better?